common/src/main/java/com/wynntils/wynn/model/map/poi/MapLocation.java

```diff
@@ -26,4 +28,17 @@ public int getY() {
     public int getZ() {
         return z;
     }
+
+    public Location asLocation() {
+        return new Location(x, y, z);
+    }
+
+    @Override
+    public String toString() {
+        return "[" + x + ", " + y + ", " + z + "]";
+    }
+
+    public String asChatCoordinates() {
+        return x + " " + y + " " + z;
```

Comment:
```suggestion
        return x + ", " + y + ", " + z;
```

Perhaps this is better?

Reply:
No, this is "chat coordinates", that is, how Vanilla expects a coordinate to be written. This way I can use vanilla functionality. 

When normally presenting to user, I have the toString method with commas.

I also intend to make a more forgiving input method, like in Legacy, for anything "coordinate like". I'm not just there yet.